portfolio/src/main/webapp/index.html

```diff
@@ -188,7 +188,10 @@ <h2>Blog</h2>
             
             <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
             </p>
-
+            
+            <button onclick="getName()">Get name!</button>
```

Comment:
Generally we ban inline event handlers since it ties our code directly into the HTML structure, among other reasons. You can attach an event handler to a button in JavaScript with addEventListener e.g.

```
myButtonElement.addEventListener('click', (e) => {
  getHello();
});
```